src/unix/linux_like/linux/mod.rs

```diff
@@ -4704,6 +4877,68 @@ extern "C" {
     ) -> ::c_int;
 }
 
+/*
+ * FIXME: when more modern kernels are available
+#[link(name = "sctp")]
+extern "C" {
```

Comment:
Could you remove them? This would introduce a maintenance burden. Also. Is it possible to cause a linking issue on a cross-compile env or something like that (i.e. a minimalist environment)? If so, I don't think the libc crate is a right place to add.

Reply:
ok fair enough.